### PR TITLE
Add tests for round tripping from CKAN => FD => CKAN

### PR DESCRIPTION
**Pull request by sglavoie**
**TL;DR**: Test against a complete CKAN package (a JSON file used as a fixture) for converting a package from CKAN to Frictionless to CKAN again. Works great in Python 3, but a better fix is needed because of incompatibility issues with Python 2.

* Making two round trips works as expected: `ckan2 == ckan3` as was the assumption. However, there are some breaking changes when Travis runs the tests for Python 2 because of Unicode issues. I haven't explored all possible solutions and added a quick fix for now (by skipping the broken tests) because the build passes on Python 3, which means the dictionaries are indeed the same when executing the `assert ckan2 == exp` statement. In Python 2, the order of the keys is not preserved and strings are dealt with differently. To fix this, we could use the same approach used in `test_extras_is_converted`, but we would need to use a smaller dictionary instead of a fixture otherwise the tests will be hard to read:

https://github.com/frictionlessdata/frictionless-ckan-mapper/blob/b97818b9cd05f5a848ef9b1de12943a2ab376150/tests/test_frictionless_to_ckan.py#L239

Because of this issue that I have tried to fix, I didn't include tests for FD => CKAN => FD as they will face the same issue. All is working on Python 3, this is only about fixing a compatibility issue on Python 2.

While I was working on those tests, I noted a few differences. I created a new issue to point those out: #33

### CODE REVIEW

File: tests/test_ckan_to_frictionless.py [hunk 1]
```diff
@@ -15,11 +19,12 @@ class TestResourceConversion:
     '''
 
     def test_fixtures(self):
+        # Test simple conversion
         inpath = 'tests/fixtures/ckan_resource.json'
         exppath = 'tests/fixtures/frictionless_resource.json'
         indict = json.load(open(inpath))
         exp = json.load(open(exppath))
-        out = converter.resource(indict)
+        out = convert_c2f.resource(indict)
```
Comment (rufuspollock) [inline tests/test_ckan_to_frictionless.py:27]: I wouldn't have done this - this test of the conversion round trip should have been a separate test since it is neither c2f or f2c but both we are testing.

This would also have obviated a load of trivial renaming in the commit.

**Comment by rufuspollock** [comment]
@sglavoie can you correct this quickly.

**Comment by sglavoie** [comment]
**FIXED**: Closing this PR in favor of #38. The issues have been addressed and there is no point in merging this one (#34), which renames too many variables.
